README.md

```diff
@@ -154,8 +154,8 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
   - [Testing](#testing)
     - [Testing Frameworks](#testing-frameworks)
     - [Mock](#mock)
-    - [Fuzzing and delta-debugging/reducing/shrinking](#fuzzing-and-delta-debuggingreducingshrinking)
-    - [Selenium and browser control tools](#selenium-and-browser-control-tools)
+  - [Fuzzing and delta-debugging/reducing/shrinking](#fuzzing-and-delta-debuggingreducingshrinking)
+  - [Selenium and browser control tools](#selenium-and-browser-control-tools)
     - [Fail injection](#fail-injection)
```

Comment:
I don't think Fail injection is a sub topic of Selenium?

Reply:
I can remove this from my PR, but the README is structured so that it is.

```
## Selenium and browser control tools

...

### Fail injection
```

Perhaps a different PR should address the headings though. I'll revert the change.